x-pack/plugins/enterprise_search/public/applications/shared/tables/reorderable_table/reorderable_table.scss

```diff
@@ -27,4 +27,8 @@
   .euiDraggable .euiDraggable__item.euiDraggable__item--isDisabled {
     cursor: unset;
   }
+
+  .unorderableRows .reorderableTableRow {
```

Comment:
question: I assume you tried, but this can't be done with attributes rather than css?

Reply:
Hmmm, I think this is fine.